shared/fs/common/index.tsx

```diff
@@ -19,4 +19,5 @@ export {
   useFsJournalStatus,
   useFsOnlineStatus,
   useFsPathInfo,
+  useFsSoftError,
```

Comment:
unused?

Reply:
I was thinking to use it to drive the Oops screen selection as well but forgot; will add it!